src/StoryContract.sol

```diff
@@ -19,29 +19,67 @@ pragma solidity 0.8.17;
 ///////////////////// IMPORTS /////////////////////
 
 import { ERC165 } from "openzeppelin/utils/introspection/ERC165.sol";
-import { IStory, NotTokenCreator, NotTokenOwner, StoryNotEnabled, TokenDoesNotExist } from "./IStory.sol";
+import { IStory } from "./IStory.sol";
+
+/*//////////////////////////////////////////////////////////////////////////
+                            Custom Errors
+//////////////////////////////////////////////////////////////////////////*/
+
+/// @dev story additions are not enabled
+error StoryNotEnabled();
+
+/// @dev token does not exist
+error TokenDoesNotExist();
+
+/// @dev caller is not the token owner
+error NotTokenOwner();
+
+/// @dev caller is not the token creator
+error NotTokenCreator();
+
+/// @dev caller is not a story admin
+error NotStoryAdmin();
```

Comment:
any reason to make these not part of the contract/interface like the events?

Reply:
mainly was for pulling in during tests - much easier than having inside the contract

Reply:
had it in the interface file but still had to be outside the interface in order to import